### PR TITLE
update docker image references quay.io

### PR DESCRIPTION
**Pull request by satyakonduri**
# Description

- Updated all the docker image references to `quay.io/dell/container-storage-modules
- Version updates

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1435|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.12.0-values.template [hunk 1]
```diff
@@ -31,7 +31,7 @@ csi-powerstore:
     podmon: 
       image: dellemc/podmon:v1.11.0
     metadataretriever: 
-      image: dellemc/csi-metadata-retriever:v1.8.0
+      image: dellemc/csi-metadata-retriever:v1.9.0
```
Comment (satyakonduri) [inline content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.12.0-values.template:34]: Path changes in installation wizard are taken care in this [PR ](https://github.com/dell/csm-docs/pull/1296/files#diff-8c916df10fa4b13e493875bfac6c8ad7658fb3040242b671f34c357c2edf73ceL34)

File: content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.12.0-values.template [hunk 2]
```diff
@@ -382,7 +382,7 @@ csi-isilon:
     authorization: 
       image: dellemc/csm-authorization-sidecar:v1.12.0
     metadataretriever: 
-      image: dellemc/csi-metadata-retriever:v1.8.0
+      image: dellemc/csi-metadata-retriever:v1.9.0
```
Comment (satyakonduri) [inline content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.12.0-values.template:385]: Path changes in installation wizard are taken care in this [PR](https://github.com/dell/csm-docs/pull/1296/files#diff-8c916df10fa4b13e493875bfac6c8ad7658fb3040242b671f34c357c2edf73ceR385)

File: content/docs/deployment/offline/modules/_index.md [hunk 3]
```diff
@@ -110,11 +110,11 @@ To perform an offline installation of a Helm chart, the following steps should b
     *
     * Loading, tagging, and pushing Docker images to registry <my-registry>:5000/
 
-      dellemc/csm-topology:v1.6.0 -> <my-registry>:5000/csm-topology:v1.6.0
-      dellemc/csm-metrics-powerflex:v1.6.0 -> <my-registry>:5000/csm-metrics-powerflex:v1.6.0
-      dellemc/csm-metrics-powerstore:v1.6.0 -> <my-registry>:5000/csm-metrics-powerstore:v1.6.0
-      dellemc/csm-metrics-powerscale:v1.3.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.3.0
-      dellemc/csm-metrics-powermax:v1.1.0 -> <my-registry>:5000/csm-metrics-powermax:v1.1.0
+      quay.io/dell/container-storage-modules/csm-topology:v1.6.0 -> <my-registry>:5000/csm-topology:v1.10.0
```
Comment (sharmilarama) [inline content/docs/deployment/offline/modules/_index.md:113]: First image needs to refer to v1.10.0 ->

File: content/docs/support/cli/_index.md [hunk 4]
```diff
@@ -845,23 +845,23 @@ dellctl images --component csi-vxflexos
...
+quay.io/dell/container-storage-modules/csi-vxflexos:v2.12.0     k8s1.31,k8s1.30,k8s1.29,ocp4.17,ocp4.16 registry.k8s.io/sig-storage/csi-attacher:v4.4.2
                                                                         registry.k8s.io/sig-storage/csi-provisioner:v3.6.2
                                                                         registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.10.0
                                                                         registry.k8s.io/sig-storage/csi-snapshotter:v6.3.2
                                                                         registry.k8s.io/sig-storage/csi-resizer:v1.9.2
                                                                         registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.1
                                                                         dellemc/sdc:4.5
 
-dellemc/csi-vxflexos:v2.8.0     k8s1.27,k8s1.26,k8s1.25,ocp4.13,ocp4.12 registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+quay.io/dell/container-storage-modules/csi-vxflexos:v2.12.0     k8s1.31,k8s1.30,k8s1.29,ocp4.17,ocp4.16 registry.k8s.io/sig-storage/csi-attacher:v4.3.0
```
Comment (sharmilarama) [inline content/docs/support/cli/_index.md:856]: Why are we repeating 2.12.0 again? @anandrajak1 aren't we only displaying latest going forward?

File: content/docs/support/cli/_index.md [hunk 5]
```diff
@@ -845,23 +845,23 @@ dellctl images --component csi-vxflexos
...
                                                                         registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
                                                                         registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.9.0
                                                                         registry.k8s.io/sig-storage/csi-snapshotter:v6.2.2
                                                                         registry.k8s.io/sig-storage/csi-resizer:v1.8.0
                                                                         registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
                                                                         dellemc/sdc:3.6.1
 
-dellemc/csi-vxflexos:v2.7.0     k8s1.27,k8s1.26,k8s1.25,ocp4.12,ocp4.11 registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+quay.io/dell/container-storage-modules/csi-vxflexos:v2.12.0     k8s1.31,k8s1.30,k8s1.29,ocp4.17,ocp4.16 registry.k8s.io/sig-storage/csi-attacher:v4.3.0
```
Comment (sharmilarama) [inline content/docs/support/cli/_index.md:864]: Same question as above.

File: content/docs/support/cli/_index.md [hunk 6]
```diff
@@ -876,19 +876,19 @@ dellctl images --component csm-authorization
...
 Driver/Module Image                             Supported Orchestrator Versions Sidecar Images
-dellemc/csm-authorization-sidecar:v1.9.0        k8s1.28,k8s1.27,k8s1.26         jetstack/cert-manager-cainjector:v1.6.1
+quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0        k8s1.31,k8s1.30,k8s1.29         jetstack/cert-manager-cainjector:v1.6.1
                                                                                 jetstack/cert-manager-controller:v1.6.1
                                                                                 jetstack/cert-manager-webhook:v1.6.1
                                                                                 ingress-nginx/controller:v1.4.0
                                                                                 ingress-nginx/kube-webhook-certgen:v20220916-gd32f8c343
 
-dellemc/csm-authorization-sidecar:v1.8.0        k8s1.27,k8s1.26,k8s1.25         jetstack/cert-manager-cainjector:v1.6.1
+quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0        k8s1.31,k8s1.30,k8s1.29         jetstack/cert-manager-cainjector:v1.6.1
```
Comment (sharmilarama) [inline content/docs/support/cli/_index.md:885]: Same question as above.

File: content/docs/support/cli/_index.md [hunk 7]
```diff
@@ -876,19 +876,19 @@ dellctl images --component csm-authorization
...
                                                                                 jetstack/cert-manager-controller:v1.6.1
                                                                                 jetstack/cert-manager-webhook:v1.6.1
                                                                                 ingress-nginx/controller:v1.4.0
                                                                                 ingress-nginx/kube-webhook-certgen:v20220916-gd32f8c343
 
-dellemc/csm-authorization-sidecar:v1.7.0        k8s1.27,k8s1.26,k8s1.25         jetstack/cert-manager-cainjector:v1.6.1
+quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0        k8s1.31,k8s1.30,k8s1.29         jetstack/cert-manager-cainjector:v1.6.1
```
Comment (sharmilarama) [inline content/docs/support/cli/_index.md:891]: Same question as above.

**Review by sharmilarama — CHANGES_REQUESTED** [review]

File: content/docs/support/cli/_index.md (hunk 4 above)
Comment (anandrajak1) [reply content/docs/support/cli/_index.md:856]: Yes, lets go with latest version alone.

File: content/docs/deployment/offline/modules/_index.md (hunk 3 above)
Comment (anandrajak1) [reply content/docs/deployment/offline/modules/_index.md:113]: Yes, lets go with latest version alone.

File: content/docs/support/cli/_index.md (hunk 7 above)
Comment (anandrajak1) [reply content/docs/support/cli/_index.md:891]: Yes, lets go with latest version alone.

File: content/docs/support/cli/_index.md (hunk 6 above)
Comment (anandrajak1) [reply content/docs/support/cli/_index.md:885]: Yes, lets go with latest version alone.

File: content/docs/support/cli/_index.md (hunk 5 above)
Comment (anandrajak1) [reply content/docs/support/cli/_index.md:864]: Yes, lets go with latest version alone.

File: hugo.toml [hunk 8]
```diff
@@ -176,19 +176,19 @@ algolia_docsearch = false
 #	icon = "fa fa-envelope"
 #        desc = "Discuss development issues around the project"
 [[params.versions]]
-  version = "Current(v1.11.0)"
+  version = "Current(v1.12.0)"
   url = "https://dell.github.io/csm-docs/docs/"
 
 [[params.versions]]
-  version = "v1.10.2"
+  version = "v1.11.0"
```
Comment (anandrajak1) [inline hugo.toml:183]: should be 1.11.1

**Review by karthikk92 — APPROVED** [review]

**Review by mjsdell — APPROVED** [review]
Approved, pending Sharmila and Anand's comments being addressed.

**Review by samihan-dell — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

File: content/docs/deployment/csmoperator/_index.md [hunk 9]
```diff
@@ -182,21 +182,21 @@ Here is the output of a request to build an offline bundle for the Dell CSM Oper
...
+   quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
+   quay.io/dell/container-storage-modules/csm-metrics-powerflex:v1.10.0
+   quay.io/dell/container-storage-modules/csm-metrics-powerscale:v1.7.0
+   quay.io/dell/container-storage-modules/csm-topology:v1.10.0
+   quay.io/dell/container-storage-modules/dell-csi-replicator:v1.10.0
+   quay.io/dell/container-storage-modules/dell-replication-controller:v1.10.0
    dellemc/sdc:4.5.2.1
-   docker.io/dellemc/dell-csm-operator:v1.7.0
+   quay.io/dell/container-storage-modules/dell-csm-operator:v1.7.0
    gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```
[the remark this answers is not in the document]
Comment (karthikk92) [reply content/docs/deployment/csmoperator/_index.md]: Addressed

**Comment by sharmilarama** [comment]
LGTM.

**Review by anandrajak1 — APPROVED** [review]

**Pull request merged**